Interview/ConductingATechnicalInterview.md

```diff
@@ -0,0 +1,24 @@
+Outline:
+
+How to Conduct a Technical Interview
+====================================
+
+### Remember what You Want to Achieve
+Best possible outcome from an interview is a new hire who makes great contributions to your organisation, your team and thrives as a member in your team. In this section I will try to say something about the mindset required to interview a candidate successfully.
+
+### Be prepared
+To not prepare for an interview is simply rude. How to best prepare does, however, depend on the kind of interview.
+
+### Make the Candidate Feel Welcome
+Discuss what we can do to make a probably nervous candidate feel welcome and respected.
+
+### Look for what the Candidate can bring
+Diverse teams are well known to be more productive but at the same time a new hire should enough about the technology we use for their intended level. How can one think about structuring an interview to learn something about what the candidate brings that we don't have already while not forgetting to ask about essential skills.
```

Comment:
> new hire should enough about

Is verb missing here? Maybe "should know enough" ?

> How can one think about structuring an interview to learn something about what the candidate brings that we don't have already while not forgetting to ask about essential skills.

I couldn't understand this English sentence... Can you simplify?

Reply:
I will try to simplify.

Reply:
I know it's something that is hard to assess with just a technical interview, but it should still be also one of the goals of an interview to understand how a Candidate can _grow_ and how open and fast they can learn new stuff and adapt. Someone that doesn't know about one of our tech/framework/architecture/patterns (e.g. RxFeedback or similar) but is a fast learner is also something interesting in a Candidate.

Reply:
@zzcgumn https://github.com/Babylonpartners/ios-playbook/pull/207#issuecomment-518736189

> The complicated thing about hiring a candidate with a great deal of experience with technologies and processes that we currently don't use is that this person is unlikely to also be an expert in the technologies we do use.

I still find this English sentence too long and hard to understand...
I believe the final documentation can be simplified even more.

For example, using "but" in the very first sentence is fading this entire section's purpose.
So, we want to split the explanation in the following way:

```suggestion
Diverse teams are well known to be more productive.
So, during the interview, blah blah...

(Below is Optional)
But at the same time, a new hire should know enough about the technology we use. 
So, blah blah... 
```